modules/core/02-client/genesis.go

```diff
@@ -65,7 +60,6 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) types.GenesisState {
 		ClientsMetadata:    clientsMetadata,
 		ClientsConsensus:   k.GetAllConsensusStates(ctx),
 		Params:             k.GetParams(ctx),
-		CreateLocalhost:    false,
```

Comment:
we might actually want to leave this param in. I'm not sure what the migration process is like for removing on chain parameters

Reply:
I've added it back in and also added it to the `AppModule` struct again (although I'm not sure if this is necessary). From what I can tell we don't actually use the field in `InitGenesis` at all.

Reply:
we can remove the `AppModule` field :+1: